### ISSUE TITLE
could you please share docker file for https://hub.docker.com/r/akadirpamukcu/tabnet

### ISSUE THREAD

**Issue by Huoran559** [DevashishPrasad/CascadeTabNet#131]
Docker always prompt error since unknown runtime specified nvidia. could you please share your docker file? then let's adapted suitable driver.  Thanks a lot :)

**Comment by joshuawnc** [comment]
Yep Yep, please share the Dockerfile, so we may all help to optimise it
Thanks a lot
